tests/sympc/module/nn/functional_test.py

```diff
@@ -41,3 +44,39 @@ def test_mse_loss(get_clients, reduction) -> None:
     res_expected = torch.nn.functional.mse_loss(y_secret, y_pred, reduction=reduction)
 
     assert np.allclose(res.reconstruct(), res_expected, atol=1e-4)
+
+
+POSSIBLE_CONFIGS_MAXPOOL_2D = [
+    (1, 1, 0),
+    (2, 1, 0),
+    (2, 1, 1),
+    (2, 2, 0),
+    (2, 2, 1),
+    (3, 1, 0),
+    (3, 1, 1),
+    (3, 2, 0),
+    (3, 2, 1),
+    (3, 3, 0),
+    (3, 3, 1),
+]
+
+
+@pytest.mark.parametrize("kernel_size, stride, padding", POSSIBLE_CONFIGS_MAXPOOL_2D)
+def test_max_pool2d(get_clients, kernel_size, stride, padding) -> None:
+    clients = get_clients(2)
```

Comment:
are we using fss_comp somewhere?

Reply:
Yes. ```max_pool2d``` needs comparison to check what is the bigger element